submitit/slurm/test_slurm.py

```diff
@@ -241,6 +241,7 @@ def test_make_sbatch_string() -> None:
         partition="learnfair",
         exclusive=True,
         additional_parameters=dict(blublu=12),
+        srun_args={"-vv": None, "--cpu-bind": "none"},
```

Comment:
looks like kwargs in the end, wouldnt it be simpler to consider them as args directly?

```suggestion
        srun_args=["-vv", "--cpu-bind none"],
```